From 98455d75df7b040817e323417b6042118c093d56 Mon Sep 17 00:00:00 2001
From: Yoann Lopes <yoann.lopes@digia.com>
Date: Thu, 31 Oct 2013 15:06:30 +0100
Subject: [PATCH] Initial porting effort to GStreamer 1.0.

* It's already included in 5.5 with this commit:
  commit 108dda7a90bd0f0337358b0db47ae55acd16dea6
  Author: Yoann Lopes <yoann.lopes@theqtcompany.com>
  Date:   Thu Nov 20 17:54:18 2014 +0100
  GStreamer: port to 1.0.
* here we restore only the done_config_gstreamer variable
  to explicitly disable it from OE build

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 qtmultimedia.pro | 26 ++++++++++++++------------
 1 file changed, 14 insertions(+), 12 deletions(-)

diff --git a/qtmultimedia.pro b/qtmultimedia.pro
index 98bb315..287887f 100644
--- a/qtmultimedia.pro
+++ b/qtmultimedia.pro
@@ -17,19 +17,21 @@ win32 {
     contains(QT_CONFIG, alsa):qtCompileTest(alsa)
     contains(QT_CONFIG, pulseaudio):qtCompileTest(pulseaudio)
 
-    isEmpty(GST_VERSION) {
-        contains(QT_CONFIG, gstreamer-0.10) {
-            GST_VERSION = 0.10
-        } else: contains(QT_CONFIG, gstreamer-1.0) {
-            GST_VERSION = 1.0
+    !done_config_gstreamer {
+        isEmpty(GST_VERSION) {
+            contains(QT_CONFIG, gstreamer-0.10) {
+                GST_VERSION = 0.10
+            } else: contains(QT_CONFIG, gstreamer-1.0) {
+                GST_VERSION = 1.0
+            }
+        }
+        cache(GST_VERSION, set)
+        !isEmpty(GST_VERSION):qtCompileTest(gstreamer) {
+            qtCompileTest(gstreamer_photography)
+            qtCompileTest(gstreamer_encodingprofiles)
+            qtCompileTest(gstreamer_appsrc)
+            qtCompileTest(linux_v4l)
         }
-    }
-    cache(GST_VERSION, set)
-    !isEmpty(GST_VERSION):qtCompileTest(gstreamer) {
-        qtCompileTest(gstreamer_photography)
-        qtCompileTest(gstreamer_encodingprofiles)
-        qtCompileTest(gstreamer_appsrc)
-        qtCompileTest(linux_v4l)
     }
 
     qtCompileTest(resourcepolicy)
